packages/hebao_v1/contracts/base/MetaTxModule.sol

```diff
@@ -137,17 +154,18 @@ contract MetaTxModule is BaseModule
         address gasToken,
         uint    gasPrice,
         uint    gasLimit,
+        uint    gasReserve,
         uint    gasSpent
         )
         private
     {
-        if (gasPrice == 0) return;
+        uint gasUsed = gasSpent + gasReserve;
```

Comment:
Maybe do `gasSpent.add(gasReserve)` just to be sure.

Reply:
done